ChemDyg_example_script.cfg

```diff
@@ -55,10 +55,11 @@ active = False
 [e3sm_diags_vs_model]
 active = False
 
-[e3sm_chem_diags]
+[chemdyg]
 active = True
-short_name = '20230221.F20TR.NGD_v3atm_v2.1oi_alt'
-years = "1985:1999:15",
+short_name = '20230307.amip.NGD_v3atm.chem-mam5-vbs.chrysalis'
+years = "2013:2013:1",
+reference_data_path = /lcrc/group/acme/public_html/diagnostics/observations/Atm/ChemDyg_inputs
```

Comment:
I would not provide the `reference_data_path` as a config option at all.  I would use `mache` to figure that out, as I will demonstrate as soon as I am able.